experimenter/experimenter/jetstream/tests/constants.py

```diff
@@ -1,6 +1,7 @@
 from copy import deepcopy
+from functools import lru_cache
```

Comment:
I mentioned this before, but I think `cache` is preferred over `lru_cache` ([link](https://docs.python.org/3/library/functools.html#functools.cache)).